antarctica_today/generate_daily_melt_file.py

```diff
@@ -71,34 +77,17 @@ def generate_new_daily_melt_files(
         # Find files in the NSIDC-0080 directory that match this date.
         nsidc_dir = tb_file_data.NSIDC_0080_file_dir
         # First fine all the .bin files that match that date stamp in the string
-        nsidc_fnames = [
-            fname
-            for fname in os.listdir(nsidc_dir)
-            if (
-                fname.find(dt.strftime("%Y%m%d")) > -1
-                and os.path.splitext(fname)[1].lower() == ".bin"
-            )
-        ]
-        # Then look specifically for the files for each frequency & polarization
-        files_37h = [
-            os.path.join(nsidc_dir, fname)
-            for fname in nsidc_fnames
-            if fname.find("s37h") > -1
+        nsidc_fps: list[Path] = [
+            fp
+            for fp in nsidc_dir.iterdir()
+            if (fp.name.find(dt.strftime("%Y%m%d")) > -1 and fp.suffix.lower() == ".nc")

```

Comment:
The files in the new version of 0080 have the different sensors available as a dimension in a single file, so we don't need to do so much work just to select files anymore.